Makefile

```diff
@@ -316,3 +316,13 @@ vectorized-bench:
 			-bench=BenchmarkVectorizedBuiltin$(VB_FILE)Func \
 			-run=BenchmarkVectorizedBuiltin$(VB_FILE)Func \
 			-args "$(VB_FUNC)"
+
+testpkg: failpoint-enable
+ifeq ("$(pkg)", "")
```

Comment:
Seems there's no definition for `$pkg`?

Reply:
See the usage `make testpkg pkg=ddl`. `pkg` variable is passed from command line

Reply:
Oh I see.